From 41fd4581940efddda0d3d9b886d8ca2bfcfb522f Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Tue, 12 May 2020 11:37:37 +0800
Subject: [PATCH] ARM: dts: rv11xx-evb-v10: adjust panel power on sequence for
 ili9881d panel

According to spec
The minimum time for VDDI/VSP on to reset high is 5ms,
The minimum time for reset to first command is 10ms.
So adjust them to save panel init time.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: Ic6c4240c1d3657d05192a5f13eb4eb469e932191
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 6a12ba49fd01..d5a0bf182d5b 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -219,12 +219,11 @@
 		reg = <0>;
 		backlight = <&backlight>;
 		power-supply = <&vcc18_lcd_n>;
-		prepare-delay-ms = <2>;
+		prepare-delay-ms = <5>;
 		reset-delay-ms = <1>;
-		init-delay-ms = <20>;
-		enable-delay-ms = <120>;
-		disable-delay-ms = <50>;
-		unprepare-delay-ms = <20>;
+		init-delay-ms = <10>;
+		disable-delay-ms = <10>;
+		unprepare-delay-ms = <5>;
 
 		width-mm = <68>;
 		height-mm = <121>;
@@ -428,8 +427,8 @@
 			15 00 02 d2 32
 			15 00 02 d3 00
 			39 00 04 ff 98 81 00
-			05 78 01 11
-			05 14 01 29
+			05 00 01 11
+			05 01 01 29
 		];
 
 		display-timings {
-- 
2.35.3

